fhir-datastore-hapi-fhir/package-metadata.json

```diff
@@ -6,6 +6,19 @@
   "version": "0.0.1",
   "dependencies": ["database-postgres"],
   "environmentVariables": {
+    "PGPOOL_IMAGE": "bitnami/pgpool:4.4.3",
+    "POSTGRES_IMAGE": "bitnami/postgresql-repmgr:14",
+    "PGPOOL_1_PLACEMENT": "node-1",
+    "PGPOOL_2_PLACEMENT": "node-2",
+    "PGPOOL_3_PLACEMENT": "node-3",
+    "PGPOOOL_REPLICAS": "1",
+    "POSTGRES_1_PLACEMENT": "node-1",
+    "POSTGRES_2_PLACEMENT": "node-2",
+    "POSTGRES_3_PLACEMENT": "node-3",
+    "POSTGRES_REPLICAS": "1",
+    "REPMGR_PRIMARY_HOST": "postgres-1",
+    "REPMGR_PARTNER_NODES": "postgres-1",
+    "REPMGR_PASSWORD": "instant101",
```

Comment:
Where are these variables going to be used? With the introduction of the Postgresdb package this won't be needed. Just recheck with the main branch.